### ISSUE TITLE
Separate copy for wipe device in bootloader mode from normal mode

### ISSUE THREAD

**Issue by matejkriz** [trezor/trezor-suite#9626]
If you click Factory reset in device settings with device in normal mode, it wipes only the seed and PIN. If you click the same button in bootloader mode, it wipes also firmware. Would be nice to have separate copy for those two variants.

**Comment by matejkriz** [comment]
requested here https://satoshilabs.slack.com/archives/C01SXSHGK44/p1696933623827499

**Comment by Hermez-cz** [comment]
### Current copy for both cases
Heading: Factory reset
Body: Factory reset wipes the device memory, erasing all information including the wallet backup and PIN. Only perform a factory reset if you have your wallet backup, which is needed to restore access to your funds.
CTA: Factory reset

### Please make separate copy for the cases as follows

**Bootloader mode**
Heading: Factory reset
Body: Performing a factory reset in bootloader mode will erase all device data, including the wallet backup, PIN, and firmware. Without your wallet backup, any funds stored on the device will be lost forever. Ensure you have your backup on hand, as it’s required to restore access to your funds after the reset. Only proceed if you’re fully prepared.
CTA: Factory reset

**Normal mode**
Heading: Wipe device
Body: Wiping the device will permanently erase all stored data, including the wallet backup and PIN. If you do not have your wallet backup, any funds on the device will be lost forever. Ensure you have your backup available before wiping the device, as it is essential for restoring access to your funds. Proceed with caution.
CTA: Wipe device

**Comment by STew790** [comment]
Should the modal be changed as well? @peter-sanderson  @Hermez-cz 

The text in settigns is correct now

Bootloader:
![Image](https://github.com/user-attachments/assets/436c15e5-ee94-4021-8b67-3e57c279a061)

Normal:
![Image](https://github.com/user-attachments/assets/8694a901-5fda-4807-b07e-cddd152a335e)

**Comment by peter-sanderson** [comment]
@STew790 Good point! I have already got the modal copy from the @Hermez-cz so I will send another PR soon.

**Comment by STew790** [comment]
**QA OK**
Ok, other than that, it LGTM.

**Comment by peter-sanderson** [comment]
@STew790 here it is: https://github.com/trezor/trezor-suite/pull/16257
